src/common/selection.c

```diff
@@ -42,7 +42,8 @@ const dt_collection_t *dt_selection_get_collection(struct dt_selection_t *select
 static void _selection_raise_signal()
 {
   // discard cached images_to_act_on list
-  darktable.view_manager->act_on.ok = FALSE;
+  dt_act_on_reset_cache(TRUE);
```

Comment:
Isn't this first call with TRUE superfluous since you call the same with FALSE below (only_visible = FALSE, means everything?).

Reply:
nope, as with this PR, we store one cache for "all images" and another one for "only visible images" so we need to reset both...
As dt_act_on_reset_cache is only called here, we could have remove the argument and always reset both caches together, but I've done it like that so we can eventually do a specific reset in some cases in the future (like a click on "group-degroup" icon for example...